graphql/resolver.go

```diff
@@ -136,6 +138,29 @@ func New(apiURL string) Config {
 			}
 		}
 
+		if operationContext == SetLastRevisionMutation {
+			projectIdentifier, ok := args["opts"].(map[string]interface{})["projectIdentifier"].(string)
```

Comment:
Can we just retrieve the project ID directly from the request or is that not possible?